jenkins/stretch.jpl

```diff
@@ -0,0 +1,13 @@
+@Library('kernelci') _
+import org.kernelci.debian.rootfs
+
+
+def k = new rootfs()
+
+def config = ['name':"stretch",
+              'arch_list':["armhf", "armel", "arm64", "i386", "amd64", "mips", "mipsel", "mips64el"],
```

Comment:
Maybe this list should be stored in the library, so when new archs get added it can be done in one place.  Can Debian arch names actually change from one release to the next?

Reply:
Archs name don't change in Debian.
The goal of listing the arches in the jenkins pipeline file is to explicitly list the archs we want to build images for. Right now, it's the same in both rootfs, but it's good to keep the flexibility to quickly disable add/archs.  And also eventually we might want to add some tests images that are not targeted to all the arches or that are build with different options depending on the target arch.